### PR TITLE
refactor: layout components

### PR DESCRIPTION
**Pull request by mainframev**
Unified same logic in `Stack`, `Inline`, `Box`
<br/><br/><br/><url>LiveURL: https://orbit-refactor-layouts.surge.sh</url>

### CODE REVIEW

File: packages/orbit-components/src/Inline/Inline.stories.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ storiesOf("Inline", module)
   .add(
     "Default",
     () => {
-      const spacing = select("spacing", SPACING, SPACING[1]);
+      const spacing = select("spacing", Object.values(SPACING), SPACING[1]);
```
Comment (vepor) [inline packages/orbit-components/src/Inline/Inline.stories.js:34]: `SPACING[1]` is `undefined`, need to use e.g. `SPACING.XXXSMALL`. It applies to all other same assertions in this file.

File: packages/orbit-components/src/Box/normalize.js [hunk 2]
```diff
@@ -50,52 +43,46 @@ const normalizeSize = (val, property): string => {
...
+    maxWidth: formatCSS("max-width", val),
+    maxHeight: formatCSS("max-height", val),
+  };
+
+  if (!all[key]) return formatCSS(key, val);
+
+  return all[key];
+};
+
 const normalize = (object: MediaQueryObject) => ({ theme }: ThemeProps) => {
```
Comment (vepor) [inline packages/orbit-components/src/Box/normalize.js:80]: Do we have tests for this normalizer? We could easily introduce bugs without them, especially if we perform refactoring 🙂

**Review by vepor — CHANGES_REQUESTED** [review]
Good job! Just some small things 🙂

File: packages/orbit-components/src/Box/normalize.js (hunk 2 above)
Comment (mainframev) [reply packages/orbit-components/src/Box/normalize.js:80]: it's covered via normal tests, no need to test normalizer, if output is correct - it means it works fine 🤔

**Review by vepor — APPROVED** [review]

**Pull request merged**